cmd/utils/flags.go

```diff
@@ -1824,6 +1824,7 @@ func SetKlayConfig(ctx *cli.Context, stack *node.Node, cfg *cn.Config) {
 		cfg.WorkerDisable = true
 	}
 
+	cfg.NoAccountCreation = !ctx.GlobalBool(ServiceChainNewAccountFlag.Name)
```

Comment:
Is this okay to be set if the node is not service chain node?

Reply:
- `SetNodeConfig()`, `SetKlayConfig()` are moved to `dumpconfigcmd.go` like the others (setDBSyncConfig, setChainDataFetcherConfig, ...) and changed as `klayConfig` receiver function.
- So that we can access other config fields like `ServiceChain.EnabledSubBridge` in `SetKlayConfig() `function [NoAccountCreation](https://github.com/klaytn/klaytn/pull/1638/files#diff-27c297ebfd606586fe73063466bc2cfffbc8a64c040b32fe3208cc41f813244dR490)

Reply:
@kjeom Hmm.. It seems a bit weird to me based on the file name. It seems all the flag setting methods were moved to `dumpcmd` file from `flags.go`. What do you think of this?

Reply:
@kjeom Understood. 
> How about putting the setting functions together in the new file, cmd/utils/config.go?

It looks better to me. I think `flags.go` can be removed then since the rest of file looks related to config, or why don't you change the file name of `flags.go` to `config.go` and put it all together?

Reply:
@jeongkyun-oh we have so many flags, so I thought we need the `flags.go` only managing the flags.

Reply:
@kjeom Alright. I agree with you.